src/web/mod.rs

```diff
@@ -429,8 +430,8 @@ fn style_css_handler(_: &mut Request) -> IronResult<Response> {
     Ok(response)
 }
 
-fn menu_js_handler(_: &mut Request) -> IronResult<Response> {
-    let mut response = Response::with((status::Ok, MENU_JS));
+fn load_js(file_path_str: &'static str) -> IronResult<Response> {
```

Comment:
Is there a reason you made this `&'static` instead of just `&'a str`?

Reply:
It definitely can be `&'a`, I just made it `&'static` because it's being passed `const` strings. It was more a semantic choice to show that we're passing known strings as opposed to, say dynamically loading JS, but I'm also happy to make it more permissive.

Reply:
Hmm actually I like this to show that it's a constant string :)